docs/csharp/tutorials/inheritance.md

```diff
@@ -220,7 +220,7 @@ The following example shows the source code for the `Publication` class, as well
 
 - Publishing-related members
 
-  Two methods, `Publish` and `GetPublicationDate`, set and return the publication date. The `Publish` method sets a private  `published` flag to `true` when it is called and assigns the date passed to it as an argument to the private `datePublished` field. The `GetPublicationDate` method returns the string "NYP" if the `published` flag is `false`, and the value of the `datePublished` field if it is `true`.
```

Comment:
This is wrong. The change corresponding to https://github.com/dotnet/docs.pt-br/pull/21 would be to change line 213 above.

And that change would still be wrong, `Title` is a get-only auto-property, it doesn't have a named backing field. So I think the whole "which stores the value in a private field named `pubTitle`" part should be removed.